public/js/main.js

```diff
@@ -1,5 +1,21 @@
-/* global $, _, Backbone */
+/* global $, _, Backbone, window, document */
 $(function () {
+
+  var tokenKey = getCookieValue('token-key')
+
+  if (tokenKey) {
+    $.ajaxSetup({
+      headers: {
+        'Authorization': window.localStorage.getItem(tokenKey)
+      },
+      error: function (x) {
+        if (x.status == 403 || x.status == 401) {
+          window.location.replace('/')
```

Comment:
This may need to be changed. If the user isn't mounting this at `/` then we shouldn't redirect them there.

Reply:
I could use `window.history.back()`, but that seems like just as bad of a solution.

What do you think about splitting the initial `GET` call into two, and use first, non-authorized request, for fetching the configuration (title, tokenKey, redirectUrl, etc.)?